Expansion for the new season - card batch #1

Pippow opened the pull request:
Hi, here is the first batch of card already approved from the main "June patch", should be ready to merge

- Monster
   - Wild Hunt
      - The Apiarian Phantom: new card, wild hunt, 2 power, deal 7, deathblow: spawn frost on the row
      - Caranthir to move 2 units to a row and apply frost
   - Fog
      - New silver The Thing in The Swamp: relict, 3pts, move any number of foglets in your graveyard to your deck and apply fog to the opposing row (could not find a better fog art)
   - Vampires- 
      - Unseen Elder 3 STR -> Lock a unit and drain it by half (just a slight buff, devs are against a full rework so at least it might make him playable)
   - Kayran: From base STR 5 - Consume a unit of 7 power or less -> base STR 4 - Consume a unit of 8 power or less
- Neutral
   - Olgiered switched to his old effect (resurect on round start and weaken by hald); current Olgiered effect moved to a new card: Olgierd Immortal
   - Lambert: Power: 8>6  Damage: 4>5
- Northern Realms
   - Cursed
      - Philippa: Mistress of the Lodge: new card, 3 points gold, mage, choose one: play a bronze or silver mage from your deck, or (chose and) play a bronze spell
      - Adda: rework to (choose) and play a bronze NR cursed unit
   - Mauler: 6 strength damage an enemy by 4, if it survives damage another by 2
- Scoiatel
   - Traps
      - Bronze trap: 11 > 12
   - Defenders
      - Add 2 armor to mahakam defenders
      - Add 1 armor to Yarpen
- Skellige
   - Discard
      - Smuggler to put back 2 bronzes
   - Self-wound
      - Svalblod Ravager or more specifically, the Raging Bear it spawns: base STR 12 - base STR 14
   - Veterans
      - Eist ability changed to spawn and play a bronze tuirseach unit (he can now spawn a veteran)
- Nilfgaard
   - Assassination: Downgrade to a silver tactic. Damage:9>7
   - Artorius Vigo: ability changed to transformed a revealed allied card into a base copy of a bronze from your starting deck
   - Amnesty now working properly

-  Descriptions have been added to Red Riders and Arjnolf, and numerous bugs have been corrected.

Pippow committed:
First Batch of new cards - code is done but the translations are WIP

Pippow committed:
Updated Translations - Ready to Merge

neal2018 commented:
This PR has git conflicts that needs resolve.

Pippow committed:
Merge branch 'diy' into SummerPatch-Batch1

Pippow commented:
> This PR has git conflicts that needs resolve.

Thanks, I dealt with it, should be good, it's the first time I resolve conflicts in Github, next time you won't need to tell me ;)

Pippow committed:
Harmonised locales files

Pippow committed:
Merge branch 'diy' into SummerPatch-Batch1

neal2018 committed:
Update cn.json

neal2018 committed:
Update cn.json

neal2018 committed:
Update cn.json

neal2018 committed:
Update cn.json

neal2018 committed:
Update cn.json

neal2018 committed:
Update cn.json

neal2018 committed:
Update GwentMap.cs

neal2018 committed:
Update GwentMap.cs

neal2018 committed:
Update GwentMap.cs

neal2018 commented on src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Monsters/Copper/Hatching.cs:
```diff
@@ -0,0 +1,36 @@
+using System.Linq;
```
I guess we need more discussion on the card "Hatching". For now, could you please remove it from the PR, so that we can merge others?

Pippow replied:
No problem.
If you feel it needs a nerf, here are 2 ideas

- Spawn 2, then 3 if there is a hatching in the graveyard
- Spawn 1 or 2, then an additional for each hatching in the graveyard

neal2018 replied:
c.c. @NPC-YFY

Pippow replied:
By the way, I would really love if in the future iterations, we could implement the changes by archetypes rather than taking a few cards from many archetypes. I mean to try to push 1 or 2 complete archetypes, as all cards here were designed to work as a package for each archetype. For instance the new "The Thing in The Swamp" serves little purpose without the additional fog sources the patch planned to introduce, or the self-wound buffs won't do much if we don't allow the units that are cursed AND damaged to count twice for the berserker marauder

neal2018 commented on src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Monsters/Silver/TheThingInTheSwamp.cs:
```diff
@@ -0,0 +1,25 @@
+                card.Effect.Repair();
+                await Game.ShowCardMove(new CardLocation(RowPosition.MyDeck, RNG.Next(0, Game.PlayersDeck[Card.PlayerIndex].Count)), card);
+            }
+            await Game.GameRowEffect[AnotherPlayer][Card.Status.CardRow.MyRowToIndex()].SetStatus<ImpenetrableFogStatus>();
```
In the card description, it first spawn fog, and then move the cards to deck, which is inconsistent with the code here. Which one is your intention?

Pippow replied:
The coded effect is the intended effect: put back any number of foglets and then spawn fog to the opposing row, it is meant to allow you to summon a foglet even there are none left in the graveyard. I updated translation files to reflect that.

neal2018 replied:
Sounds good. Thanks!

neal2018 commented on src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/NorthernRealms/Copper/AedirnianMauler.cs:
```diff
@@ -17,6 +17,16 @@ public override async Task<int> CardPlayEffect(bool isSpying, bool isReveal)
                 return 0;
             }
             await target.Effect.Damage(4, Card);
+            var TARGET = target;
```
what is this variable for? I did not see it used?

Pippow replied:
Probably some remnant of a previous iteration on my end, I removed it, thanks.

Pippow commented:
@neal2018 I did all the modifications you asked.

Pippow committed:
Followed Neal's instructions:
- removed a useless variable from mauler's code
- removed "hatching" until further discussion
- corrected "The Thing In The Swamp"'s descritpion to match effect

neal2018 committed:
Update GwentMap.cs

Pippow commented:
> I have modified some details and translations.
> 
> It should be ready to merge.
> 
> c.c.@NPC-YFY

Thanks a lot! And well done on spotting the missing faction tag on "The Thing In The Swamp", I think I deleted that by mistake when fixing the conflicts. And thanks for fixing the Chinese, I had to use automated translator. Excited to finally see this come through!

Mattia900 committed:
Fix EN cards descriptions (#437)

Co-authored-by: neal <46561493+neal2018@users.noreply.github.com>

cattermore committed:
Update ru.json (#457)

cattermore committed:
Update pl.json (#456)

neal2018 committed:
feat(Artorius Vigo): nerf (#455)

neal2018 committed:
Merge branch 'diy' into tmp
